tests/spec/fs.rename.spec.js

```diff
@@ -69,6 +69,16 @@ describe('fs.rename', function() {
       });
     });
   });
+  //337
+  it('should error if the file or directory to rename doesnt exist', function(done) {
+    var fs = util.fs();
+
+      fs.rename('/fakeDirectory', '/myotherdir', function(error) {
+        expect(error).to.exist;
+        done();
```

Comment:
Great Job! @itchylol742 
I just want to add one my opinion that if you want to test the existing of a path/file,
it might be a good way to add:

expect(error.code).to.equal('ENOENT'); 

Thank you :)